src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -74,4 +74,8 @@ public boolean equals(Object other) {
                 || (other instanceof AddCommand // instanceof handles nulls
                 && toAdd.equals(((AddCommand) other).toAdd));
     }
+    @Override
+    public String toString() {
+        return "Add Person Command";
```

Comment:
If you included the `Person` details here (eg. maybe even just the `Name`), could it be possible to include more details in the `UndoCommand` success other than just `Add Person Command undone`?

Same for the other commands that have specific details (eg. `Delete` and `Edit`), so the user is more clear on what in particular has been undone.

Reply:
I'll give it a shot